.github/workflows/batch_release_pr.yml

```diff
@@ -0,0 +1,40 @@
+name: "Creates Batch Release for A Package"
+
+on:
+  repository_dispatch:
+    types: [batch_release_pr]
+
+jobs:
+  create_release_pr:
+    runs-on: ubuntu-latest
+    env:
+      BRANCH_NAME: ${{ github.event.client_payload.package }}-${{ github.run_id }}-${{ github.run_attempt }}
+    steps:
+      - name: checkout repository
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8
+      - name: Set up tools
+        run: dart pub get
+        working-directory: ${{ github.workspace }}/script/tool
+      - name: create batch release PR
+        run: |
+          git config --global user.name ${{ secrets.USER_NAME }}
+          git config --global user.email ${{ secrets.USER_EMAIL }}
+          dart ./script/tool/lib/src/main.dart branch-for-batch-release --packages=${{ github.event.client_payload.package }} --branch=${{ env.BRANCH_NAME }} --remote=origin
+      - name: Check if branch was created
```

Comment:
This could use a comment explaining that `branch-for-batch-release` will not create a branch if there's nothing to release, as context for why this is here.